ui/pages/onboarding-flow/secure-your-wallet/skip-srp-backup-popover.js

```diff
@@ -91,12 +90,11 @@ export default function SkipSRPBackup({ handleClose }) {
           className="skip-srp-backup-popover__icon"
           color={IconColor.errorDefault}
         />
-        <Typography
-          variant={TypographyVariant.H3}
-          fontWeight={FONT_WEIGHT.BOLD}
+        <Text
+          variant={TextVariant.headingMdBold}
```

Comment:
`TextVariant.headingMdBold` doesn't exist, Devin should either be using the fontWeight prop or be abled to discern that headingMd is already bold 
```suggestion
          variant={TextVariant.headingMd}
```

<img width="974" alt="Screenshot 2024-05-31 at 4 59 34 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/0f63a31b-b610-45b2-ae03-2a4938461688">